ipywidgets_bokeh/widget.py

```diff
@@ -5,24 +5,42 @@
 # The full license is in the file LICENSE.txt, distributed with this software.
 #-----------------------------------------------------------------------------
 
-from bokeh.core.properties import Any
+from bokeh.core.properties import Any, String
 from bokeh.models.layouts import HTMLBox
 
 from ipywidgets import embed, Widget
 
-from .kernel import kernel
```

Comment:
Why this line was removed?

Reply:
Ahh, sorry about that - I think I must have removed it to get rid of the flake8 warning that `kernel` wasn't being used. I'll replace it now.

Reply:
I figured that was the case.